models/issues/label.go

```diff
@@ -87,6 +87,7 @@ type Label struct {
 	OrgID           int64 `xorm:"INDEX"`
 	Name            string
 	Exclusive       bool
+	ExclusiveOrder  int `xorm:"DEFAULT 0"`
```

Comment:
A migration is required if a new column added to a table.

Reply:
Added